src/core/datatypes/trees/Clade.cpp

```diff
@@ -168,6 +175,11 @@ bool Clade::operator<(const Clade &c) const
         return false;
     }
     
+    if ( c.clade_name != clade_name )
+    {
+        return c.clade_name < clade_name;
+    }
+    
     for (size_t i = 0; i < taxa.size(); ++i)
```

Comment:
do we want clades with the same members but different names to be unequal?

Reply:
Same as above. Probably not necessary and therefore commented out now.